### ISSUE TITLE
The loss decreases too fast using LCSTS dataset

### ISSUE THREAD

**Issue by lixiang1991** [abisee/pointer-generator#44]
Hi~
Using the CNN/DailyMail dataset this system works well.
Then I just change dataset, create .bin files with make_datafiles.py and remain the other parameters, but the training log shows that the loss decrease sharply. So I try to set the learning rate 0.001, and train again. The loss decrease from 12 to 11 in 547 steps. At step 1000 the loss is 8.17,  at step 1200 it becomes 1.16, then decrease to 0.0005 at 4000 step.
There is no doubt that it will not has correct answer.
I have checked the batch.enc_batch_extend_vocab, it prints like below
```
[[   334   1382   6242    103     24   3296    241    296     62 150000
      64   1114    221     58     16     15     93      5    154   1015
    2707     59    101    177    235      4   5387 150000      4    221
    1204     58     16   1212      5  84537     14     10    277 150001
    5830    186     59      4 150002   5830  16287    184    396]
 [  4485     15    173      9    136    972   5124  41986   5382    939
    6361   6728  26273    316   7177   3164   2650    674    309    230
    6326     11    348     12    390    114   9415      5  11653    774
    2840     21    309    230      4    364   1299     87  41986   5382
     939   2074    358    258    110   4051   2761   7177  25514]
 [  1566     39    621      6    121   8799    938   1606    292   1606
     939   3582   8799     21   8983     26   1057  20809  40671  20809
      68   1606   7928     26    939   3582     21    192    236  20809
      68   1606     26    729   1650      4    939   3582   8799   2238
    5583      4     81   5009    505    568     48      4   1263]
 [   456     16  29598      4   3803   5274     20      5   1710     99
    3088      4   6316    176  19774     62  10862     64      5     70
     566    392     96   2879   7705    501   3507     16   3088   5694
       5   9442  10862   8560      8   5967   2879    384  15864   1050
       4   6416   9533   2300   9533   6416  59298    356  47838]
 [ 14514    452    320     12   5783   1257   3639      6    357  35592
   32166     23      4  19297  29786   3848   1078   1135     36    301
     685    161   1544      4   1257    211     20    793     31    479
       4  40736    517     12  10463   1164    127   2891    388   1715
    1164   8408   5210    762   4600   1569   4444 150000    301]
 [  1019   1063    434    681     66      4   2255    487   1401    248
    4487  59742   1063    441   3686   7584    226  59742   1063    441
       4   1130     20      7  10823    710  77964   1063     17   4198
       4   2935   3047   3472    175    177    676    140  11373    139
      68     23   1019   1063   1019    236    434   8777   3803]
 [ 12389   4032  17556   1450     19  38773   2259   2646    482    260
      19     98   8161   4383      6     28   1209  31204    186   1194
    8161  42849  38502  78383      5  19652     32     51   1319   8282
     303    642    278    439     69  21486  12801   2181      6    245
      27    511     50   8161     20  10496    818   6951   1698]
 [  1925    350     34    113     35  33334   2750  17948      6 150000
       4  12936     42     86  17948      4    381  11519  23592   3947
   11599      4   7147   2348     65   8162    699   2056   2310   1925
    3402    508  10814   4392   1652   2675    611   1246    113  38183
      10     70  27439      4    235  34064   6406    429   6338]
 [   176     16   1623  11907   3379    159    295   6207  10247   7443
     738 150000    398     25     56   2890   1944    154     37   7497
     489    423    214   5483   4443    324   5111    302    295   3339
    1508   4018 150001   5368   1918    388    122    295   7443      9
   12074  26006    340     39   1155   5368   1135    505  23609]
 [  8136     56   1623   3089     10    123   5348    359    238   5151
      18     27   8136 150000    202     10    219   2369   1653     15
      66     55     18   8136    591      8   2704    597   1296     18
      27   8136   1783      8   2383    902    971    208   8719    503
      18   8136   2424      8   3268   1033   4440  11870     18]
 [     5   6855   5112     14      4     55  11585     23    977    198
    3255   1288  15077    516    474    666    884      4    386 150000
     977   1519    607    330   4050     57  15077    516  22281    531
     789   2474  12443   7899   5668     21   7070    889    659  10469
     203    977    303   3492  15077    516      4    757  99748]
 [   348     12   1734   1541    719     56   5737   1494    723   3970
    4090  58354   6589  23431 150000   2523     31  38817    338      9
    3923 150001  11233     10  37245   2009    137  22433  33885   1237
    1141   6148    137   1827    278  13465  17452    338     19    919
    5737    880  21829   2203     99    201   4833  90369  11937]
 [   589    266   1447    787    959    192    109   1818    102    583
     101   1176    589   1447    313   3346    360    890   2971    192
     548     62  19867    474     64      4    589   2022  43774   2022
      35     67  15800   2333   7477 143086  25228  87109   1889  92349
   87639   5926   1142  23269     21    139   3194     67  10848]
 [  8189     12   1453    244     10    223   7012    998   1113    316
      37   7497     75    159      4    135    146   4712   5114   1013
    1627   3021     88  34247     47   8426   1696    414   4079  27565
    2953  23567    169    159     29   1602     26  10247   1141   1173
   51509     21    973     49   2724    312    127   4045  74211]
 [   181    174     19 139373     35   3444  10828  12818   1756   8959
       5  12315     13   1325   1151   4796   7656   1693    170    808
    1151   5013     39   1353    665  77126    978    895   4796    258
     113   2029  25221   5232    343  11900    587   4725    333  20412
    1101     98  73789      4   4796    604     81   1153     28]
 [    75    136 150000    120    469    434      5    438   4879    924
   11788   4714   3530      4    121    120  11836    438      4  42018
       9  13785      4   1948     89      9   2861    561   8419    438
    1948   3971     42     10  65627   6520      6  12615   3246    987
    3246      5    391   1163   3006   2743   1196    136   5460]]
```
where vocab size is 150k.
So maybe the inputs have no problem?
I guess only changing the learning rate will not help to train a correct model.
Any suggestion will be appreciated.

**Comment by wutingkang** [comment]
hi~
I also use LCSTS to train this model. After word segmentation, vocab size = 50000, learning rate=0.1, is fine, but without participle and vocab size=5000, learning rate =0.1/0.02/0.01 ,loss drops very fast, just like yours. Have you solved the problem?

**Comment by Balabala125** [comment]
您好，我现在也在学习文本摘要，想在中文文本摘要上尝试一下，但是由于中文文摘的数据集太少了，所以我想请问一下，您可不可以给我分享一下您的数据集呀？真的非常感谢！！

**Comment by wutingkang** [comment]
中文数据集参考（由于是非公开数据集，需要的请向拥有着申请）：
http://www.sogou.com/labs/resource/cs.php
http://icrc.hitsz.edu.cn/Article/show/139.html

**Comment by Balabala125** [comment]
好的，非常感谢

**Comment by Balabala125** [comment]
您好，我还想请教一下，你是怎么把LCSTS数据处理成可以输入到网络模型中的bin文件的呢？（我尝试了一段时间，总是各种出错）

**Comment by wutingkang** [comment]
可以参考这个：
https://github.com/yangzhiye/Short-Text-Summarization/blob/7e1134b876d678ea883264778d42e214e1662ceb/get_data.py
和这个：
https://blog.csdn.net/hfutdog/article/details/78447860

**Comment by Balabala125** [comment]
好的，再次感谢！！！
